synapse/storage/state.py

```diff
@@ -541,7 +541,7 @@ async def get_state_ids_for_event(
             state_filter: The state filter used to fetch state from the database.
 
         Returns:
-            A dict from (type, state_key) -> state_event
+            A dict from (type, state_key) -> state_event_id
```

Comment:
Updated the return comment for `get_state_ids_for_event` to be correct.

It looks like a typo copy/paste from `get_state_for_event` above.

Reply:
Split out to https://github.com/matrix-org/synapse/pull/10050